### PR TITLE
#128 Bugfix - Get annotation by id

### PR DESCRIPTION
**Pull request by laila1222**
- Commented out moduler lines that throw error
- Modified controller and router for 'getting annotation by id'
- Added comments with the endpoint paths

### CODE REVIEW

**Comment by OlkaA** [comment]
Nice work 👍

**Review by OlkaA — APPROVED** [review]

File: src/server/api/controllers/annotations.controller.js [hunk 1]
```diff
@@ -30,22 +29,14 @@ const getAnnotations = async () => {
...
-        'title',
-        'description',
-        'fk_screenshot_id',
-        'area',
-        'updated_at',
-        'created_at',
-        'deleted_at',
-      )
-      .where({ id });
+      .where({ annotation_id: annotation_id })
```
Comment (bytewiz) [inline src/server/api/controllers/annotations.controller.js]: You could write this line out as `.where({ annotation_id })` if you want to save space 👍

File: src/server/api/routes/api-router.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const express = require('express');
 const router = express.Router();
 
 // Router imports
-const modulesRouter = require('./modules.router');
+// const modulesRouter = require('./modules.router');
```
Comment (bytewiz) [inline src/server/api/routes/api-router.js]: Just remove the commented out code :v:

File: src/server/api/routes/api-router.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const swaggerUi = require('swagger-ui-express');
 router.use('/documentation', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
 // Application routes
-router.use('/modules', modulesRouter);
+// router.use('/modules', modulesRouter);
```
Comment (bytewiz) [inline src/server/api/routes/api-router.js]: Just remove the commented out code :v:

**Review by bytewiz — CHANGES_REQUESTED** [review]
Very minor requests looks good 👍 just some out commented code we don't need 🥇

File: src/server/api/controllers/annotations.controller.js (hunk 1 above)
Comment (laila1222) [reply src/server/api/controllers/annotations.controller.js]: done

File: src/server/api/routes/api-router.js (hunk 2 above)
Comment (laila1222) [reply src/server/api/routes/api-router.js]: done

File: src/server/api/routes/api-router.js (hunk 3 above)
Comment (laila1222) [reply src/server/api/routes/api-router.js]: done

**Comment by zkwsk** [comment]
You need to resolve some conflicts.

**Review by bytewiz — APPROVED** [review]
Looks good to me 👍

**Pull request merged**